README.md

```diff
@@ -1,6 +1,7 @@
-# Google Student Training in Engineering Program
+# Google Student Training in Engineering Program 
```

Comment:
nit: try to avoid committing unnecessary whitespace changes, as that makes viewing the history of changes harder